### PR TITLE
Major urgent changes please approve ASAP

### PR DESCRIPTION
**Pull request by fabiovincenzi**

### CODE REVIEW

File: src/main/java/HelloWorld.java [hunk 1]
```diff
@@ -2,4 +2,7 @@ public class HelloWorld {
     public static void main(String[] args) {
 	System.out.println("Hello World!");
     }
+    public static void printClassName() {
```
Comment (S-furi) [inline src/main/java/HelloWorld.java:9]: I would like some javadoc, because this method seems a bit confusing to me 🗿

File: src/main/java/Main.java
Comment (S-furi) [inline src/main/java/Main.java:1]: Very good class mate!

**Review by S-furi — CHANGES_REQUESTED** [review]
plz check instrcutions asap ! 11 !

File: src/main/java/HelloWorld.java (hunk 1 above)
Comment (fabiovincenzi) [reply src/main/java/HelloWorld.java:9]: done can you please resolve 🙏

**Review by S-furi — APPROVED** [review]
Very Good@!

**Pull request merged**